csp_test.go

```diff
@@ -180,6 +180,18 @@ func TestCSP(t *testing.T) {
 			html:   `<img src="https://blah.com" />`,
 			valid:  false,
 		},
+		{
+			policy: "img-src 'self'",
+			page:   "https://google.com",
+			html:   `<link rel="icon" href="https://blah.com" />`,
+			valid:  false,
```

Comment:
Would be nice to have a positive case here as well

Reply:
Done